tests/mocha/field_angle_test.js

```diff
@@ -201,10 +201,14 @@ suite('Angle Fields', function() {
       this.angleField.htmlInput_ = Object.create(null);
       this.angleField.htmlInput_.oldValue_ = '1';
       this.angleField.htmlInput_.untypedDefaultValue_ = 1;
+      this.stub = sinon.stub(this.angleField, 'resizeEditor_');
```

Comment:
Why do you need to stub this?  The FieldAngle shouldn't be rendered, so resizeEditor shouldn't be called.

Reply:
``onHtmlInputChange_`` is called, which in turn now calls ``resizeEditor_``